src/Command/PruneEsLogsCommand.php

```diff
@@ -21,7 +21,7 @@ class PruneEsLogsCommand extends Command
     public function __construct(
         private readonly Client $client,
         private readonly Configuration $configuration,
-        private readonly Builder $builder
+        private readonly QueryBuilder $builder
     ) {
```

Comment:
Add a nullable logger here in case and call `->error` on it in the failure states below as needed